python/scipp.cpp

```diff
@@ -22,12 +22,12 @@ PYBIND11_MODULE(_scipp, m) {
   m.attr("__version__") = py::str("unknown version");
 #endif
 
+  init_units_neutron(m);
   init_dataset(m);
   init_dimensions(m);
   init_dtype(m);
   init_neutron(m);
   init_sparse_container(m);
-  init_units_neutron(m);
```

Comment:
Did the order matter here?

Reply:
yes the order mattered, as `make_variable.h` needed to know about the units. It was fine when it was only used in `variable.cpp` because `init_units_neutron` was performed before `init_variable`